wormhole/wormhole.go

```diff
@@ -63,8 +63,8 @@ var (
 	// DefaultRendezvousURL is the default Rendezvous server to use.
 	DefaultRendezvousURL = "ws://relay.magic-wormhole.io:4000/v1"
 
-	// DefaultTransitRelayAddress is the default transit server to ues.
-	DefaultTransitRelayAddress = "transit.magic-wormhole.io:4001"
+	// DefaultTransitRelayURL is the default transit server to ues.
+	DefaultTransitRelayURL = "tcp:transit.magic-wormhole.io:4001"
```

Comment:
These are breaking changes, which would require us to bump the major version number of the module. I'd really prefer to make these changes in a backwards compatible way.

I would suggest we add the new fields `TransitRelayURL` and `DefaultTransitRelayURL` but leave the existing fields and mark them as deprecated. Then in the code, if the old fields are set their values would be used, otherwise use the new fields. 

This strategy would be similar to how `net/http` has handled deprecated fields.